test/solidity/assignments/assignments.js

```diff
@@ -0,0 +1,29 @@
+const { expect } = require('chai')
```

Comment:
Add license header

Reply:
We dont have license headers in the other js files?